gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/task/EhrExtractMessageHandler.java

```diff
@@ -262,6 +287,29 @@ private PatientAttachmentLog buildPatientAttachmentLogFromAttachment(
     private PatientAttachmentLog buildPatientAttachmentLogFromExternalAttachment(
```

Comment:
We _shouldn't_ need this behaviour for an `ExternalAttachment` (based on the example we've been given within the ticket).